cmake/thirdparty/FindSLEPc.cmake

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2024, Lawrence Livermore National Security, LLC. All rights
+# reserved.  LLNL-CODE-856946
+
+# OFFICIAL USE ONLY This work was produced at the Lawrence Livermore National
+# Laboratory (LLNL) under contract no. DE-AC52-07NA27344 (Contract 44) between
+# the U.S. Department of Energy (DOE) and Lawrence Livermore National Security,
+# LLC (LLNS) for the operation of LLNL.  See license for disclaimers, notice of
+# U.S. Government Rights and license terms and conditions.
```

Comment:
```suggestion
# Copyright (c) 2019-2024, Lawrence Livermore National Security, LLC and
# other Serac Project Developers. See the top-level LICENSE file for
# details.
#
# SPDX-License-Identifier: (BSD-3-Clause)

```

Reply:
Unless this is a direct copy/paste from another code, it should have our copyright. If it did, then we should provide attribution.

Reply:
Hm, I originally wrote it for LiDO. Should I just use Serac's?